eng/common/scripts/common.ps1

```diff
@@ -20,12 +20,43 @@ $EngScriptsLanguageSettings = Join-path $EngScriptsDir "Language-Settings.ps1"
 if (Test-Path $EngScriptsLanguageSettings) {
   . $EngScriptsLanguageSettings
 }
-If ($LanguageShort -eq $null)
+if ($null -eq $LanguageShort)
 {
   $LangaugeShort = $Language
 }
+if ($null -eq $isDevOpsRun)
+{
+  $isDevOpsRun = ($null -ne $env:SYSTEM_TEAMPROJECTID)
+}
 
 # Transformed Functions
 $GetPackageInfoFromRepoFn = "Get-${Language}-PackageInfoFromRepo"
 $GetPackageInfoFromPackageFileFn = "Get-${Language}-PackageInfoFromPackageFile"
-$PublishGithubIODocsFn = "Publish-${Language}-GithubIODocs"
\ No newline at end of file
+$PublishGithubIODocsFn = "Publish-${Language}-GithubIODocs"
+
+function LogHelper ($logType, $logArgs)
```

Comment:
Can you please put this and the isDevOpsRun variable into their own script and just import it here? I think these might be useful in some other standalone cases as well.